pkg/server/billing.go

```diff
@@ -141,8 +139,54 @@ type BillingHandlerFunc func(
 ) ([]*billingpb.Record, int64)
 
 var handlers = map[string]BillingHandlerFunc{
-	"cpu": handleCPUBilling,
-	"ram": handleRAMBilling,
+	"cpu":       handleCPUBilling,
+	"ram":       handleRAMBilling,
+	"ip":        handleIPBilling,
+	"drive_ssd": handleSSDDriveBilling,
+	"drive_hdd": handleHDDDriveBilling,
+}
+
+var handleSSDDriveBilling BillingHandlerFunc = makeDriveBillingHandler("SSD")
```

Comment:
Driver type can be any(hdd, ssd, nvme, joemama), thus handler must be dynamic and universal